web/src/engine/osk/src/specialCharacters.ts

```diff
@@ -40,9 +40,32 @@ let specialCharacters = {
   '*RTLBkSp*':        0x72,
   '*ShiftLock*':      0x73,
   '*ShiftedLock*':    0x74,
-  '*ZWNJ*':           0x75, // If this one is specified, auto-detection will kick in.
+  '*ZWNJ*':           0x79, // If this one is specified, auto-detection will kick in.
```

Comment:
@jahorton - I updated this to match the codepoint from the font, but left in the KeymanWeb code that overrides to use the iOS/Android version.

Reply:
At the very least, as is, we'd get a desync with the following:

https://github.com/keymanapp/keyman/blob/5455aa741b8394ec0b63531b7c6fe8fe3b072c0c/developer/src/tike/xml/layoutbuilder/constants.js#L594

@mcdurdin - would this cause impacts on the Developer side?  Looking at `CSpecialText10` and `CSpecialText14`, I see something that could be interpreted as a pseudoarray in the defined strings... and if that's used to compute an "index" of sorts to reproduce the code, the change here would also cause a breakage there.

Reply:
I think we need to stick with 0x75 to match existing behaviour. We could add a new `'*ZWNJGeneric*': 0x79` (which matches the naming style of `*ZWNJiOS*` and `*ZWNJAndroid*`) to specifically use the generic one.

We do need to update the Developer side of this as well so the keyboard devs can pick the new names.

Reply:
> We do need to update the Developer side of this as well...

Started in another branch
https://github.com/keymanapp/keyman/tree/chore/developer/non-printing-chars

Both of these are more back-burner tasks that I started during the planning week.

Reply:
Reverted ZWNJ
Added ZWNJGeneric
And rebuilt the test keyboard

Reply:
Since the IDs are different now...

https://github.com/keymanapp/keyman/blob/4d2dd477a655ef14bac48d01d4c7759fa312b7aa/web/src/engine/osk/src/keyboard-layout/oskKey.ts#L285-L301

Might want to add `case '*ZWNJGeneric*':` as an alternate case matching the same block (as `case '*ZWNJ*':`) of the switch-case statement.

Reply:
> I suppose that keeping them separate would make sense.

Yes, that's why I suggested the separate identifier.